backend/config/services.php

```diff
@@ -36,5 +38,12 @@
             ['name' => 'console_api_resource', 'priority' => 150]
         );
 
+    // ================ HEALTH CHECKS =================
+    $services->set(AllQueuesHaveAtLeastOneIpHealthCheck::class)
```

Comment:
global level configs like these are always a bad idea. Instead of using a Service config, get the classes in  the runAllHealthChecks method